tests/test_helper/setup_teardown/tls_cluster.bash

```diff
@@ -0,0 +1,31 @@
+setup_file() {
+    load test_helper/common.bash
+    load test_helper/lxd.bash
+    load test_helper/microovn.bash
+
+
+    TEST_CONTAINERS=$(container_names "$BATS_TEST_FILENAME" 4)
+    CENTRAL_CONTAINERS=""
+    CHASSIS_CONTAINERS=""
+    counter=0
+    # Categorize first three containers as "CENTRAL" and reset as "CHASSIS"
+    for container in $TEST_CONTAINERS; do
+        if [ $counter -lt 3 ]; then
+            CENTRAL_CONTAINERS+="$container "
+        else
+            CHASSIS_CONTAINERS+="$container "
+        fi
+        ((++counter))
+    done
+
+    export TEST_CONTAINERS
+    export CENTRAL_CONTAINERS
+    export CHASSIS_CONTAINERS
+    launch_containers jammy "${TEST_CONTAINERS[@]}"
```

Comment:
Note that I'm in the process of cleaning up the erroneous use of array references to this variable, so might as well do that here as well, ref:
https://github.com/fnordahl/microovn/commit/f08136b4520c0139ac806aa0dbac9e2897beabbb

Reply:
Done